js_modules/dagit/packages/core/src/workspace/useRepositoryForRun.ts

```diff
@@ -7,7 +7,7 @@ import {findRepoContainingPipeline, repoContainsPipeline} from './findRepoContai
 
 type MatchType = {
   match: DagsterRepoOption;
-  type: 'origin-and-snapshot' | 'origin-only' | 'snapshot-only' | 'pipeline-name-only';
+  type: 'origin-only' | 'pipeline-name-only';
```

Comment:
Change this `'origin' | 'pipeline-name-only'`? Since the implication is now that `origin` means that it's a "complete" match, including the job name.